packages/carton-git/carton-git.0.1.0/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "Implementation of PACK file in OCaml"
+description: """\
+Carton is an implementation of the PACK file
+in OCaml. PACK file is used by Git to store Git objects. Carton is more
+abstracted when it can store any objects."""
+maintainer: "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors: "Romain Calascibetta <romain.calascibetta@gmail.com>"
+license: "MIT"
+homepage: "https://github.com/dinosaure/carton"
+doc: "https://dinosaure.github.io/carton/"
+bug-reports: "https://github.com/dinosaure/carton/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.6.0"}
+  "carton"
+  "carton-lwt"
+  "bigarray-compat"
+  "bigstringaf"
+  "lwt"
+  "fpath"
+  "result"
+  "mmap"
+  "fmt" {>= "0.8.7"}
+  "base-unix"
+  "decompress" {>= "1.2.0"}
+  "astring" {>= "0.8.5"}
+  "alcotest" {>= "1.2.3" & with-test}
+  "alcotest-lwt" {>= "1.2.3" & with-test}
+  "cstruct" {>= "6.0.0" & with-test}
+  "logs" {>= "0.7.0" & with-test}
+  "mirage-flow" {>= "2.0.1" & with-test}
+  "rresult" {>= "0.6.0" & with-test}
+  "ke" {>= "0.4" & with-test}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+dev-repo: "git+https://github.com/dinosaure/carton.git"
+x-commit-hash: "d7d3ddfccf0aa2c3f273481ec2ef5b84f3a4d054"
+url {
+  src:
+    "https://github.com/dinosaure/carton/releases/download/carton-v0.1.0/carton-carton-v0.1.0.tbz"
```

Comment:
Is it expected for `carton-git` and `carton-lwt` to have a different archive from `carton`? (`dinosaure/carton` vs. `mirage/ocaml-git`)

Reply:
Seems related to the OPAM file, I will fix that.